assignments/prototypes.js

```diff
@@ -32,7 +50,20 @@
   * should inherit destroy() from GameObject through CharacterStats
   * should inherit takeDamage() from CharacterStats
 */
- 
+function Humanoid (beast) {
+  this.faction = beast.faction;
+  this.weapons = beast.weapons;
+  this.language = beast.language;
+  CharacterStats.call(this, beast);
+} 
+Humanoid.prototype = Object.create(GameObject.prototype);
```

Comment:
We only need to inherit from `CharacterStats`, it will automatically inherit `GameObject` as well through the inheritance chain.